domain.yml

```diff
@@ -183,6 +185,12 @@ responses:
   - text: Ég hætti þá leitinni og við getum byrjað upp á nýtt ef þú vilt. Afsakaðu vesenið.
   utter_validation_failed_fallback:
   - text: Því miður skil ég þetta ekki. Ég verð að hætta leitinni og byrja upp á nýtt.
+  utter_joke:
+  - text: Hvað borða vegan vampírur? Blóðappelsínur.
+  - text: Hvað gera köngulær þegar þeim leiðist? Hanga á vefnum!
+  - text: Hvað verður um ís sem er búinn að vera lengi í boxi? Hann kemst á endanum í form.
+  utter_operator:
```

Comment:
Maybe not for today, but I thought of the joke intent as an example of an intent where the system uses an external file to retrieve answers. In a real use case there would be many more jokes (or other standardized answers to choose from randomly, Sæmi e.g. knows "málshættir" and "nýyrði Jónasar Hallgrímssonar"), so maintaining such a list in the code base is not feasible